content/business-models/none.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "None"
+date: 2019-01-09
+weight: 1
+---
+
+This is typically the ur-form of sustainable Free and Open Source Communities.
+The business model is to *not have a business model*. That is okay, because
+not every sustainable open source project needs or wants a business model.
+
+In this case the project has set up means and rules of collaboration, has a
+formal membership, usually without a formal position of authority or any other
+hierarchy but does **not** deal with monetary contributions.
+
+Sustainability in this non-model focuses on collaborator growth (quality and
+quantity) and shared responsibility to address it's biggest risk: contributor
```

Comment:
spelling: _it's_ is short for "it is" and in this case should be _its_
`quantity) and shared responsibility to address its biggest risk: contributor`